src/pybind/cephfs/setup.py

```diff
@@ -72,7 +72,7 @@ def get_python_flags(libs):
         libraries=libs + py_libs,
         extra_compile_args=filter_unsupported_flags(
             compiler.compiler[0],
-            distutils.sysconfig.get_config_var('CFLAGS').split()),
+            compiler.compiler[1:] + distutils.sysconfig.get_config_var('CFLAGS').split()),
```

Comment:
ACK: [On platforms with a command-line (Unix, DOS/Windows), each of these is a string that will be split into executable name and (optional) list of arguments](https://docs.python.org/2/distutils/apiref.html#distutils.ccompiler.CCompiler.set_executables).